src/bench.ts

```diff
@@ -21,6 +19,7 @@ import {
 } from './constants'
 import { createBenchEvent } from './event'
 import { Task } from './task'
+import { pLimit } from './utils'
```

Comment:
nit: lets also not call this `pLimit`

its a concurrency helper. e.g. `withConcurrency` or some such thing

this isn't a fork of p-limit or even associated with it